.husky/pre-commit

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
-
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+#
+#export NVM_DIR="$HOME/.nvm"
+#[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
❓ I'm not sure about this, can someone clarify why this is needed ? The workflow doesn't work with this lines

Reply:
From https://github.com/camunda/camunda-docs/commit/2b5c57c1e2bab6b74c2b678b10e67ae68379e387#r88648535: "let's see if anyone complains" -- it took almost two years, but it finally happened! 😅 

I think these lines were snuck in for a dev environment that was running nvm, and it's intended to _not_ cause problems for someone that isn't. @nicpuppa can you point me at a failed workflow, so I can try to identify if this script can be rewritten to support its intended use, and also work for this workflow? I won't spend a lot of time investigating, and if I can't get anywhere, then I support the removal of these lines.

Reply:
This is the [failed workflow](https://github.com/camunda/camunda-docs/actions/runs/10563015584/job/29262273110). 
This is the [husky issue](https://github.com/typicode/husky/issues/1176) that point me to this removal

Reply:
From that linked issue:

>  it was a minor version difference for node. Specifically he was running node 16.17 and the .nvmrc file for the project specified 16.13. 

Looking closer, this seems likely the culprit here, too. I didn't expect it, but [the image used by the workflow](https://github.com/actions/runner-images/blob/ubuntu22/20240818.1/images/ubuntu/Ubuntu2204-Readme.md) does include nvm! I would have expected it not to be there, in which case that line wouldn't have executed anything. 

But it does, and since [our .nvmrc](https://github.com/camunda/camunda-docs/blob/main/.nvmrc) hasn't been updated in a while, it's likely using a version that isn't installed on the workflow instance. 

The couple of solutions I can think of are: 
1. remove these lines. This might mean that some contributors need to update their shell profile so that nvm is loaded there, instead of in this pre-commit hook. (Unless I misunderstand why it was originally introduced, this is likely what should have been done all along.)
2. Update our .nvmrc to a more recent version (maybe even just the most recent 16, not sure which versions are installed on the workflow image). I don't really want to do this right now, because it feels like it could explode, and is definitely out of scope of this PR.

All this to say, I support removing these lines. @akeller @nicpuppa do you have a preference whether we remove them in this PR, or whether I open a separate PR since it is outside the scope of this PR?

Reply:
You can remove them in this PR so we have the context all in the same PR.